oat-sa/extension-tao-delivery
# Feature/actp 298/main layout header replace header menu

**Related**
https://oat-sa.atlassian.net/browse/ACTP-298

**Description**:
The header can't be replaced by a customer theme

**How to test**
1. Add the template with `delivery-header` key
2. Login to the tao as a test taker
3. The new template should be displayed on the delivery and test runner pages
